components/ipfs/translate_ipfs_uri.cc

```diff
@@ -42,9 +42,7 @@ bool TranslateIPFSURI(const GURL& url, GURL* new_url, bool local) {
       // new_url would be:
       // https://dweb.link/ipfs/[cid]//wiki/Vincent_van_Gogh.html
       if (new_url) {
-        std::string gateway_empty_path =
-            local ? GetDefaultIPFSLocalGateway().spec()
-                  : GetDefaultIPFSGateway().spec();
+        std::string gateway_empty_path = gateway_url.spec();
         if (gateway_empty_path.size() > 0 &&
             gateway_empty_path[gateway_empty_path.size() - 1] == '/') {
```

Comment:
not part of this PR, but couldn't we use `GURL::Resolve()` for these manipulations?